pipeline/experiment.py

```diff
@@ -0,0 +1,346 @@
+import datajoint as dj
+import numpy as np
+
+from . import lab
+from . import get_schema_name
+
+schema = dj.schema(get_schema_name('experiment'))
+
+
+@schema
+class Session(dj.Manual):
+    definition = """
+    -> lab.Subject
+    session: smallint 		# session number
+    ---
+    session_date: date
+    session_time: time
+    unique index (subject_id, session_date, session_time)
+    -> lab.Person
+    -> lab.Rig
+    """
+
+
+@schema
+class Task(dj.Lookup):
+    definition = """
+    # Type of tasks
+    task            : varchar(12)                  # task type
+    ----
+    task_description : varchar(4000)
+    """
+    contents = [
+        ('audio delay', 'auditory delayed response task (2AFC)'),
+        ('audio mem', 'auditory working memory task'),
+        ('s1 stim', 'S1 photostimulation task (2AFC)'),
+        ('foraging', 'foraging task based on Bari-Cohen 2019'),
+        ('foraging 3lp', 'foraging task based on Bari-Cohen 2019 with variable delay period')
+    ]
```

Comment:
replace contents with 
```suggestion
    contents = [
        ('fm texture discrim', 'freely moving texture discrimination)'),
        ('hf texture discrim', 'head fixed texture discrimination)'),
        ('hf pole loc', 'head fixed pole localization)'),
        ('hf wall dist', 'head-fixed wall distance'),
        ('hf wheel', 'head-fixed wheel runing')
        ('gen tun', 'unstructured generic tuning stimulation)')
    ]
```
@ksseverson57 For Wenxi's task, does this work ?  ('hf wall dist', 'head-fixed wall distance')
other task suggestions?